back/Gemfile.lock

```diff
@@ -353,6 +353,7 @@ PATH
   specs:
     multi_tenancy (0.1.0)
       actionpack-cloudfront (~> 1.2.0)
+      nlp
```

Comment:
This change should be accepted.

See [this Slack thread](https://citizenlabco.slack.com/archives/C016C2EHURY/p1658224834942099) for more context.